File: app/src/main/java/com/app/bickup_user/retrofit/ApiUtils.java
package com.app.bickup_user.retrofit;

import com.app.bickup_user.utility.ConstantValues;

public class ApiUtils {
    private ApiUtils() {}
    public static APIService getAPIService() {
 
        return RetrofitClient.getClient(ConstantValues.BASE_URL).create(APIService.class);
    }
}

File: app/src/main/java/com/app/bickup_user/restservices/ApiUtils.kt
package com.app.bickup_user.restservices

import com.app.bickup_user.restservices.Constant.BASE_URL
import com.app.bickup_user.restservices.Constant.BASE_URL2

object ApiUtils {

        val apiService: APIService
        get() = RetrofitClient().getClient(BASE_URL)!!.create(APIService::class.java)

        val apiService2: APIService
        get() = RetrofitClient().getClient(BASE_URL2)!!.create(APIService::class.java)
}